from typing import final


try:
    num = 15/1
    print(num)
except : 
    print('Something went to wrong')
finally:
    print('this is done')